api/v1alpha1/openapi.yaml

```diff
@@ -279,13 +279,6 @@ components:
         id:
           type: string
           format: uuid
-        name:
```

Comment:
Maybe in future, we can gather vcenter name or something and display it, so it's recognized somehow in UI?